templates/vsftpd.conf.erb

```diff
@@ -24,82 +24,51 @@ local_umask=<%= @local_umask %>
 # Uncomment this to allow the anonymous FTP user to upload files. This only
 # has an effect if the above global write enable is activated. Also, you will
 # obviously need to create a directory writable by the FTP user.
-<% if @anon_upload_enable == 'YES' -%>
```

Comment:
Mentioned change is according to what we have on current ftp/sftp production server.